### DISCUSSION TITLE
What is your favorite country? / 好きな・訪れたい国はどこですか？

### DISCUSSION THREAD

**Discussion by makoto-u** [NAIST-SE/AttractingContributors#172] [General]
Please tell me about a country you would like to visit someday or a country you like.
Thanks for your contribution!
https://github.com/makoto-u/AttractingContributors-Favorite-Country

**Comment by tktkbohshi** [comment]
added!
plz help me...
https://github.com/tktkbohshi/AttractingContributors-favorite-niconico-douga

**Reply by makoto-u** [reply]
Thanks.
I've added 'Battlefield 2042 Official Reveal Trailer' to your list.

**Comment by t-matsumoto4758** [comment]
Added!
please help me!
https://github.com/t-matsumoto4758/AttractingContributors-YourHometown

**Reply by makoto-u** [reply]
Thanks.
I've added 'Miyagi' to your list.

**Comment by xincanfeng** [comment]
added.
can you help by telling me your favirate color?
https://github.com/FengXincan/AttractingContributors-color
